nix/inferno-ml/migrations/v1-create-tables.sql

```diff
@@ -42,6 +42,10 @@ create table if not exists mversions
     -- that saving a model version requires as its first step `lo_import`ing
     -- the contents
   , contents oid not null
+    -- The size of the `contents` above in bytes. This could be calculated
+    -- on-demand, but since the `contents` are immutable, we can calculate
+    -- this once and store it here
+  , size bigint not null
```

Comment:
Unfortunately an `integer` column is probably not going to be big enough, as it could only represent ~2Gb in size and we might have things bigger than that